tests/integration/testdata/start_api/terraform/terraform-v2-api-simple/main.tf

```diff
@@ -22,27 +28,27 @@ EOF
 }
 
 resource "aws_s3_bucket" "lambda_code_bucket" {
-  bucket = "lambda_code_bucket"
+  bucket = "lambda_code_bucket_${random_uuid.unique_id.result}"
 }
 
 resource "aws_s3_object" "s3_lambda_code" {
-  bucket = "lambda_code_bucket"
+  bucket = "lambda_code_bucket_${random_uuid.unique_id.result}"
   key    = "s3_lambda_code_key"
   source = "HelloWorldFunction.zip"
 }
 
 resource "aws_lambda_function" "HelloWorldFunction" {
-  s3_bucket     = "lambda_code_bucket"
+  s3_bucket     = "lambda_code_bucket_${random_uuid.unique_id.result}"
   s3_key        = "s3_lambda_code_key"
   handler       = "app.lambda_handler"
   runtime       = "python3.8"
-  function_name = "HelloWorldFunction"
+  function_name = "HelloWorldFunction_${random_uuid.unique_id.result}"
   timeout       = 500
   role          = aws_iam_role.iam_for_lambda.arn
 }
 
 resource "aws_apigatewayv2_api" "my_api" {
-  name           = "my_api"
+  name           = "my_api_${random_uuid.unique_id.result}"
   protocol_type  = "HTTP"
 }
 
```

Comment:
for resource `"aws_apigatewayv2_route" "example"` .. I think the `target` property value should be in format `"integrations/${<integration id>}"`. If it is working using this format when we run `terraform apply`, please add another example to use the previous format as explained in [the doc](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/apigatewayv2_route)